### PR TITLE
NEXTEUROPA-11865 CLONE - EC resp - caption for image doesn't support translations

### PR DESCRIPTION
**Pull request by ademarco**

### CODE REVIEW

**Comment by Fefaine** [comment]
:+1:

File: tests/features/images.feature [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    And I attach the file "/tests/files/logo.png" to "edit-field-image-und-0-upload"
+    And I press "Save"
+    Then the response should contain "has been created."
+    When I go to "/admin/content/file"
+    And I click "logo.png"
+    And I click "Edit"
+    And I fill in "Name" with "Image name"
+    And I fill in "Caption" with "Image caption"
+    And I press "Save"
+    And the response should contain "has been updated."
```
Comment (cyberwolf) [inline tests/features/images.feature]: Then ...

**Comment by voidtek** [comment]
:+1:

**Comment by ademarco** [comment]
👎

**Comment by ademarco** [comment]
I would put the test back (since it does test what I have fixed) and defer the discussion to another issue.

**Comment by Fefaine** [comment]
:+1:

File: profiles/common/themes/ec_resp/templates/fields/field--image.tpl.php [hunk 2]
```diff
@@ -46,16 +46,18 @@
 ?>
 <div class="<?php print $classes; ?>"<?php print $attributes; ?>>
   <?php if (!$label_hidden): ?>
-    <div class="field-label"<?php print $title_attributes; ?>><?php print $label ?>:&nbsp;</div>
+    <div
```
Comment (netlooker) [inline profiles/common/themes/ec_resp/templates/fields/field--image.tpl.php]: Is there a reason why formating of this tag was changed?

File: profiles/common/themes/ec_resp/templates/fields/field--image.tpl.php [hunk 3]
```diff
@@ -46,16 +46,18 @@
...
+      class="field-label"<?php print $title_attributes; ?>><?php print $label ?>
+      :&nbsp;</div>
   <?php endif; ?>
   <div class="field-items"<?php print $content_attributes; ?>>
     <?php foreach ($items as $delta => $item): ?>
-      <div class="field-item <?php print $delta % 2 ? 'odd' : 'even'; ?>"<?php print $item_attributes[$delta]; ?>>
-      <?php print render($item); ?>
+      <div
```
Comment (netlooker) [inline profiles/common/themes/ec_resp/templates/fields/field--image.tpl.php]: Similar comment like for line 49.

File: profiles/common/themes/ec_resp/template.php [hunk 4]
```diff
@@ -347,11 +349,11 @@ function ec_resp_preprocess_node(&$variables) {
     $variables['revision_name'] = theme('username', array('account' => $revision_account));
     $variables['revision_date'] = format_date($node->changed);
     $variables['submitted'] .= "<br />" . t('Last modified by !revision-name on !revision-date',
-      array(
-        '!revision-name' => $variables['revision_name'],
-        '!revision-date' => $variables['revision_date'],
-      )
-    );
+        array(
```
Comment (netlooker) [inline profiles/common/themes/ec_resp/template.php]: Previous indentations were correct.

**Pull request merged**